openvdb_houdini/houdini/GEO_VDBTranslator.cc

```diff
@@ -70,12 +70,16 @@
 #include <UT/UT_IStream.h>
 #include <UT/UT_Version.h>
 
+#include <FS/FS_IStreamDevice.h>
```

Comment:
My apologies....  I should have broken this into two pull requests for exactly that reason.  I tried to avoid creating specific topic branches, but that then meant my second request got implicitly merged with the first :<  If you can get to the #ifdefs before me, I'll integrate your changes.  The fileStat() should work in 15.0, just the stream-aliasing is specific to 16.0  Otherwise I'll make a new update next week.  Sorry again for this situation.